app/lib/client/client.dart

```diff
@@ -303,8 +303,8 @@ class SPHclient {
       } else {
         throw CredentialsIncompleteException();
       }
-    } catch (e, stack) {
-      recordError(e, stack);
+    } catch (e) {
+      if (e is CredentialsIncompleteException) rethrow;
       throw LoggedOffOrUnknownException();
     }
```

Comment:
```suggestion
    } on CredentialsIncompleteException {
      rethrow;
    } catch (e) {
      throw LoggedOffOrUnknownException();
    }
```
Statt mit `is` zu überprüfen einfach weiteren Catch-Phrase hinzufügen